packages/dd-trace/test/appsec/rasp/ssrf.express.plugin.spec.js

```diff
@@ -162,6 +162,49 @@ describe('RASP - ssrf', () => {
           })
         })
       })
+
+      describe('Test using request', () => {
+        withVersions('express', 'request', requestVersion => {
+          let requestToTest
+
+          beforeEach(() => {
+            requestToTest = require(`../../../../../versions/request@${requestVersion}`).get()
+          })
+
+          it('Should not detect threat', async () => {
+            app = (req, res) => {
+              requestToTest.get(`https://${req.query.host}`).on('response', () => {
+                res.end('end')
+              })
+            }
+
+            axios.get('/?host=www.datadoghq.com')
+
+            return checkRaspExecutedAndNotThreat(agent)
+          })
+
+          it('Should detect threat doing a GET request', async () => {
```

Comment:
did you make sure this test was failing before you implemented the fix ?

Reply:
yes

![image](https://github.com/user-attachments/assets/0ab9c49b-aa28-496a-b1a1-4cb308f9e471)